packages/memory-inspector/src/browser/diff-widget/memory-diff-table-widget.tsx

```diff
@@ -16,7 +16,7 @@
 
 import { inject, injectable } from '@theia/core/shared/inversify';
 import * as React from '@theia/core/shared/react';
-import * as Long from 'long';
+import Long from 'long';
```

Comment:
These are necessary because of the scanoss changes. Because we consume the scanoss library, we get new versions of `long` etc. in which require these changes.